### ISSUE TITLE
Tree Beacon Doesn't Work With Dynamic or Aether Trees

### ISSUE THREAD

**Issue by MaxAstro** [thephoenixlodge/Heavens-of-Sorcery#34]
As the title says... I was excited to finally be able to make a tree beacon and set up an automatic farm, but the tree beacon doesn't work with dynamic trees or skywood trees, so there doesn't seem to be any trees you can plant that it actually does work with.  :(

**Comment by Birgliss** [comment]
while this is correct (the tree beacon does not work with dynamic trees), a workaround for now is to craft the tree seeds into saplings with a dirt bucket. this is a conversion recipe offered by dynamic trees in the case of incompatibility

**Comment by MaxAstro** [comment]
Yeah, I tried that.  Imagine my frustration when the sapling turns into a dynamic tree as soon as I plant it. XD

**Comment by RobertLSnead** [comment]
in the dynamic trees config file change the to the following rule to use vanilla style saps instead of automatic conversion, which in turn will allow you to use the bucket as stated above and it will work just fine.

vanilla {
    # Right clicking with a vanilla sapling places a dynamic sapling instead. [default: false]
    B:replaceVanillaSapling=false
}

**Comment by MaxAstro** [comment]
Thank you!

**Comment by gattsuru** [comment]
0.9.6.1, (most) Blue Sky trees, including Starlit Trees, can be turned into Charcoal.  Maple's the only one I've been having problems getting rid of since.

**Comment by ewanm89** [comment]
Try using rannucarpus to plant the vanilla sappling, or a dispensor, or anything but a player.

**Comment by BuggaNoia** [comment]
> 
> 
> Try using rannucarpus to plant the vanilla sappling, or a dispensor, or anything but a player.

Bonemeal on Dispenser placed sapling gives dynamic tree :(
